contracts/epochs/FlowDKG.cdc

```diff
@@ -216,8 +219,20 @@ pub contract FlowDKG {
 
             // if a key is different, stop checking this submission
             // and return false
-            if key != existingSubmission[index] {
-                return false
+            if let keyUnwrap = key {
```

Comment:
I would expect the way you had it previously would still work. Cadence seems to handle equality of optional strings the way you would expect based on a simple [example in the Playground](https://play.onflow.org/aadabdd1-4a25-46ce-86ca-865c2178361d?type=script&id=9d8b8c96-7e4c-450b-95a5-cb7e6ec0c694).

Reply:
good point! I'll change it back